src/pages/Transaction/Title.tsx

```diff
@@ -16,23 +16,23 @@ export default function TransactionTitle({transaction}: TransactionTitleProps) {
 
   return inDev ? (
     <Stack
-      direction={{xs: "column", md: "row"}}
+      direction={{xs: "column"}}
       alignItems="flex-start"
       spacing={2}
       marginX={1}
       marginBottom={2}
     >
       {isOnMobile ? (
         <>
-          <Stack direction="row" alignItems="center" spacing={2}>
-            <Typography variant="h5">Transaction:</Typography>
+          <Stack direction="column" alignItems="center" spacing={2}>
+            <Typography variant="h3">Transaction</Typography>
```

Comment:
I think we want to have the same title variant for all pages? I'll change for the rest pages in a follow up PR.